docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,422 @@
+_output/local/bin/edgecore --edgeclusterconfig > /etc/kubeedge/config/edgecore.yaml
+tests/edgecluster/hack/update_edgecore_config.sh [cluster_B_kubeconfig_file]
+```
+
+
+# 2	Run Fornax Cluster in Machine B, C
+## summary
+- Use tmux and split window. Run following sequence step.
+- Step 1: machine A cloud core
+- Step 2: machine B edge core
+- Step 3: machine B cloud core
+- Step 4: machine C edge core
+- Step 5: back to machine A second window to run deployment command and other command(kubectl get edgecluster, kubectl get mission)
+- Each *Mission* can be deployed to multiple clusters, so the status is a collection of all workloads from the same *Mission*.
+
+##  2.1.	In machine A.
+- If you have tmux on your machine, split two window view. Otherwise start two command window. One window run the cloud-core, One window check cluster status
+### 2.1.1. One window run following cloudcore command line (notes: machine A only run cloudcore)(Step 1):
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/cloudcore
+
+```
+Result:
+<img src="images/Running_A_cloudcore.png" 
+     width="98%"  
+     align="center"/>
+     
+### 2.1.2 Another window run following line, to check machine status and mission deployment status (Step 5)
+```
+kubectl get crd
+kubectl get edgecluster
+kubectl get mission
+```
+
+##  2.2.	In machine B. (Notes: If we have C machine, we need also run "cloudcore" in machine B.)
+### 2.2.1 Run edgecore in machine B (Step 2)
+- following command line only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/edgecore --edgecluster
+```
+- Notes: If you first time to run above command, please run the following command. only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+
+### 2.2.2. Run cloudcore in machine B (if you have machine C) (Step 3)
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/cloudcore
+```
+Result:
+<img src="images/Running_B_edgecoreAndcloudcore.png" 
+     width="98%"  
+     align="center"/>
+
+##  2.3.	In machine C. (only run edgecore)
+### 2.3.1 Run edgecore in machine C (Step 4)
+- following command line only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+```
+export KUBECONFIG=/etc/kubernetes/admin.conf
+_output/local/bin/edgecore --edgecluster
+```
+- Notes: If you first time to run above command, please run the following command. only run one time.
+```
+chmod 777 /root/go/src/github.com/kubeedge/_output/local/bin/kubectl/vanilla/kubectl
+```
+
+Result:
+<img src="images/Running_C_edgecore.png" 
+     width="98%"  
+     align="center"/>
+
+# 3	Deployment Mission to Machine B, C
+## 3.1. Waiting machine A, B, C running, we can run following command and test mission deployment
```

Comment:
where's the "following command"?

Reply:
updated: ## 3.1. Waiting machine A, B, C running, we can start 3.2